docs/website/content/docs/v0.1/Getting Started/resources.md

```diff
@@ -0,0 +1,119 @@
+---
+description: ""
+weight: 4
```

Comment:
Looks like `concepts.md` is weighted at the same weight.

Reply:
Concepts goes away as part of this, as everything that was covered there was a CRD. So this will take its place.

Reply:
Ah I missed that.